### PR TITLE
esn-frontend-inbox#53: Brought back the Linshare upload provider in the mail composer

### PR DESCRIPTION
**Pull request by huy-ta**
Partially resolves https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/53. Please also refer to these 2 PRs:

- https://github.com/OpenPaaS-Suite/esn-frontend-inbox-linshare/pull/3
- https://github.com/OpenPaaS-Suite/esn-frontend-linshare/pull/4

Demo (desktop):

Demo-Linshare-Uploader-Desktop

Demo (mobile):

Demo-Linshare-Uploader-Mobile

Note: The dynamic directives' removal will be handled in another issue (https://github.com/OpenPaaS-Suite/esn-frontend-inbox/issues/60).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,8 +30,8 @@
     "babel-plugin-angularjs-annotate": "^0.10.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.6.0",
-    "esn-frontend-login": "github:openpaas-suite/esn-frontend-login#main",
     "dotenv-webpack": "^2.0.0",
+    "esn-frontend-login": "github:openpaas-suite/esn-frontend-login#main",
```
Comment (tuanlc) [inline package.json:34]: This does not belong to the commit

Comment (huy-ta) [reply package.json:34]: I used `npm install`, so it auto-rearranged the order of the packages. I think it's fine to keep it this way.
